examples/linksdkreact/src/App.tsx

```diff
@@ -0,0 +1,77 @@
+import { CodatLink } from "https://link-sdk.codat.io";
+import { useState, useEffect } from "react";
+
+import logo from "./logo.svg";
+import "./App.css";
+
+function App() {
+  const [companyId, setCompanyId] = useState("");
+  const [appOpen, setAppOpen] = useState(false);
+  const [componentMount, setComponentMount] = useState<HTMLDivElement | null>(
+    null
+  );
+
+  useEffect(() => {
+    const target = componentMount;
+    if (target && target.children.length === 0) {
+      new CodatLink({
+        target,
+        props: {
+          companyId,
+          onConnection: (connection) =>
+            alert(`On connection callback - ${connection}`),
+          onClose: () => setAppOpen(false),
+          onFinish: () => alert("On finish callback!!"),
+          onError: (error) => alert(`On error callback -${error}`),
+          baseUrl: "https://link-api.codat.io",
+        },
+      });
+    }
+  }, [componentMount, companyId]);
+
+  return (
+    <div className="App">
+      <h1>
+        <a
+          className="App-link"
+          target="_blank"
+          href="https://github.com/codatio/codat-docs/blob/embedded-link-enhancements/docs/auth-flow/authorize-embedded-link.md#get-started-with-react"
+        >
+          Link SDK X Create React App
+        </a>
+      </h1>
+      <div className="link-wrapper">
+        <ol>
+          <li>
+            <a className="App-link" href="#" target="_blank">
```

Comment:
https://github.com/codatio/sdk-link/blob/main/examples/react/README.md

Reply:
will have to link to a guide to create a company via api/portal rather than repo